### PR TITLE
Add PaymentSheet support for Twint

### PR DESCRIPTION
**Pull request by adityar-stripe**
## Summary
Adds Payment Sheet support for Twint

## Motivation
- Add Twint support

## Testing
Screenshot 2024-02-21 at 18 37 57
Screenshot 2024-02-21 at 18 38 09

Manual testing 

## Changelog
No line in changelog since this is a Private feature still in Pilot phase.

### CODE REVIEW

**Comment by adityar-stripe** [comment]
r?

File: StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodEnums.swift [hunk 1]
```diff
@@ -86,6 +86,8 @@ import Foundation
     case promptPay
     /// A Swish payment method
     case swish
+    /// A TWINT payment method
+    case twint
```
Comment (yuki-stripe) [inline StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodEnums.swift:90]: Can you add a test in `PaymentSheet_LPM_ConfirmFlowTests`?

**Review by yuki-stripe — COMMENTED** [review]
Nice!

**Comment by adityar-stripe** [comment]
Added the test r? @yuki-stripe

**Review by yuki-stripe — APPROVED** [review]

**Pull request merged**